recipes/sapiens/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "sapiens" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 41a8242da8ced4c4390ab6167950c6abb53ce84e3a7519282ccd680254d3119a
+
+build:
```

Comment:
It looks like this project has an `entry_point` for a console script. It should be added. See:
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points

and then a simple test of the command line tool should be added to the test section:
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-commands

Reply:
Oh right, I created the entrypoint in setup.py, but it doesn't actually exist. So I deleted it from setup.py for now.